### ISSUE TITLE
 Could not find trigger "XXXX" in your functions directory

### ISSUE THREAD

**Issue by ruanjianusa** [vercel/next.js#9135]
# Examples bug report
Everything seems to work except when Firebase serving the webapp, it can't find the resource properly.  

## Example name
with-firebase-hosting
with-firebase-hosting-and-typescript

Provide the example name

## Describe the bug
Followed exactly as example, when "npm run serve" , no error print in terminal.
open URL as prompted, errors occur as follow:
1) in terminal (where its run): 
⚠  functions: Could not find trigger "zWsSEbvS8S3oz12Qpis_4" in your functions directory.
⚠  Your function was killed because it raised an unhandled error.
⚠  functions: Could not find trigger "runtime" in your functions directory.
⚠  Your function was killed because it raised an unhandled error. 
2) in Browser Console: [multiple times]
Failed to load resource: the server responded with a status of 500 (Internal Server Error)

## To Reproduce
This error happen both in my code base and when I construct a new example.

Simply do
a. npx create-next-app --example with-firebase-hosting-and-typescript with-firebase-hosting-and-typescript
cd with-firebase-hosting-and-typescriptnpm run serve

Then you can see the err messages

## System information

- OS: MacOS
- Browser: chrome
- Version of Next.js: lastest

**Comment by timneutkens** [comment]
@jthegedus

**Comment by jthegedus** [comment]
Apologies for the delay. I hope you found a more immediate solution to your issue and haven't been blocked by this since. With improvements in my availability I will be back to lend a hand in Firebase issues.

@ruanjianusa Are you still experiencing this problem and if so, can you provide a link to a reproduction repo for me to look at? Cheers.

**Comment by moaaz-bhnas** [comment]
I have the same issue

**Comment by pururaj1908** [comment]
@timneutkens Can I take this issue as my first contribution?

**Comment by camunoz2** [comment]
Hi! I wanted to work on this, but dont find any errors in Google Chrome, or Firefox or Edge,

I followed this steps

- Created my firebase app in the [web console](https://console.firebase.google.com/)
- Then i login trought the cli (sfter installing the firebase tools) with
```bash
firebase login
```
- Then i listed the projectId with
```bash
firebase projects:list
```
- Then installed the template with
```bash
npx create-next-app --example with-firebase-hosting testing
```
- Then copied the id inside the `.firebaserc` default field
- Then after running
```bash
npm run serve
```
- Go to localhost:5000 or localhost:4000
- Checking the terminal or the browser console, don't print any errors related to the post.

**Comment by balazsorban44** [comment]
Hi, since the issue is quite old, and the above comment suggested it works, I decided to close the issue.

We welcome contributions to https://github.com/vercel/next.js/tree/canary/examples/with-firebase-hosting (See https://github.com/vercel/next.js/tree/canary/contributing/examples as a guide).

If you want to work on this feel free to open a PR and we can review there, or open a new issue if you have a different problem with the [example](https://github.com/vercel/next.js/tree/canary/examples/with-firebase-hosting).

Thanks!
